packages/grid/_modules_/grid/components/toolbar/ExportSelector.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import MenuList from '@material-ui/core/MenuList';
+import Button from '@material-ui/core/Button';
+import MenuItem from '@material-ui/core/MenuItem';
+import { ApiContext } from '../api-context';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridMenu } from '../menu/GridMenu';
+import { ExportOption } from '../../models';
+
+export function ExportSelector() {
```

Comment:
To move one step in the right direction, we should prefix it with `GridToolbar`.
I'm not sure `Selector` as a suffix adds much, we could remove it.

Reply:
No preference about the `Selector` suffix. I added it because of the `DensitySelector` and `ColumnSelector`. I'll update the export name.